<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>After_30_days</class>
 <widget class="QDialog" name="After_30_days">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>prescription after 30 days</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>date of issue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>how many packages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="how_many_pac"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>the smallest pack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="small_pac"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>packaging size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="size_pac"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>dosage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dosage"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
